docs/guide/browser/index.md

```diff
@@ -511,7 +511,7 @@ import { html } from 'lit'
 import './greeter-button'
 
 test('greeting appears on click', async () => {
-  const screen = render(html`<greeter-button name="World"></greeter-button>`)
+  const screen = await render(html`<greeter-button name="World"></greeter-button>`)
```

Comment:
This is `vitest-browser-lit` example

The react examples are in https://github.com/vitest-dev/vitest/blob/c0d4ece81555dd839a9d901fc3e671c4f4f1ae3b/packages/vitest/src/create/browser/creator.ts

Reply:
🤦🏻‍♂️